include/blogWithComments_functions.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+$Error = array();
```

Comment:
I noticed that elsewhere you have an `$Errorz` function. Is that so it doesn't have the same name as this? You should be able to have a single common `$Error` array that you can use for any form. Talk to me about this if you want to know how.